catalogs/catalog-lakehouse-generic/src/main/java/org/apache/gravitino/catalog/lakehouse/lance/LanceTableOperations.java

```diff
@@ -100,45 +109,59 @@ public Table createTable(
     String location = properties.get(Table.PROPERTY_LOCATION);
     Preconditions.checkArgument(
         StringUtils.isNotBlank(location), "Table location must be specified");
-    Map<String, String> storageProps = LancePropertiesUtils.getLanceStorageOptions(properties);
+
+    // Extract creation mode from properties
+    CreationMode mode =
+        Optional.ofNullable(properties.get(LANCE_CREATION_MODE))
+            .map(
+                modeStr -> {
+                  try {
+                    return CreationMode.valueOf(modeStr);
+                  } catch (IllegalArgumentException e) {
+                    throw new IllegalArgumentException(
```

Comment:
I'm not very sure about the intention that `catch IllegalArgumentException and then throw an IllegalArgumentException`